src/Program.cs

```diff
@@ -32,6 +44,8 @@ public static class Program
             "Content-Type",
         };
 
+        private static readonly string _responseSelectionHeader = "x-ms-faultinjector-response-option";
```

Comment:
```suggestion
        private const string _responseSelectionHeader = "x-ms-faultinjector-response-option";
```

NIT: Prefer `const` over `static readonly`.